_posts/guides/2023-04-26-guide_adding_slick_to_your_map.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Adding Slick to Your Map
+permalink: /guide/adding-slick-to-your-map/
+category: guide
+tags:
+  - slick
+  - defrag
+toc: false
+toc_sticky: false
+---
+
+Slick is a defrag mechanic that gives frictionless ground movement. Slick is distinct from the slide mechanic that [`func_slide`](/entity/func_slide/) provides. There are three ways to add slick to a map: the [`func_slick`](/entity/func_slick/) brush entity, the `%compileSlick` material paramater, and the `SetSlick` player input.
+
+# `func_slick`
+
+The [`func_slick`](/entity/func_slick/) entity is a brush entity that will give the slick effect to the player when they stand on it. This entity has no special properties.
+
+{:.notice--info}
+This entity is always forced to use BSP collision, overriding any mapper-set value.
+
+# `%compileSlick` Material Property
+
+Any material with the `%compileSlick` parameter set to 1 will act as a slick surface when compiled onto world geometry or `func_detail`.
+
+{:.notice--info}
+A slick material is only guaranteed to work on world geometry or `func_detail`, it is not known to work on other brush entities, displacements, or props.
+
+Here is an example VMT for a slick material:
+
+```
+"LightmappedGeneric"
+{
+    $basetexture test_materials\slick_ice
+    %compileSlick 1
+}
+```
+
+# `SetSlick` Player Input
```

Comment:
Do we have a general page for player inputs? It might make this info easier to find.

Reply:
No, but it might be nice in the future to have an entire section for them, like with cvars and entities.